.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,16 @@
+### Issues
+
+What issues are being solved?
+
+### Todos
+
+What's outstanding?
+
+### Impacted Areas
+
+What components will be affected?
+
+### Steps to Reproduce
+
+What do we need to run, or what do we need to do to test this?
+
```

Comment:
@misscs Do you think we could use the `Issues` section for that? Or do you envision creating a separate one?